api-gateway/src/main/resources/static/scripts/bill-details/bill-details.template.html

```diff
@@ -38,6 +38,9 @@ <h5>Address: 505 Lincoln Str. Queens, New York</h5>
             <h5>Bill Status: <span ng-style="{'color': $ctrl.bills.billStatus === 'OVERDUE' ? 'red' : 'black'}">{{$ctrl.bills.billStatus}}</span></h5>
             <hr>
             <br>
+            <h5>Due Date: <strong>{{$ctrl.bills.dueDate}}</strong></h5>
```

Comment:
I like the use of <strong> as it makes the due date "pop" more which is good because the due date is important!